packages/contracts/evm-contracts/contracts/token/InverseAppProjectedNft.sol

```diff
@@ -61,7 +61,7 @@ contract InverseAppProjectedNft is IInverseAppProjectedNft, ERC721, Ownable {
             super.supportsInterface(interfaceId);
     }
 
-    function currentNonce(address user) external view returns (uint256) {
+    function currentNonce(address user) public view returns (uint256) {
```

Comment:
This no longer matches the interface

Reply:
Not true, functions in interfaces **must** be declared external, implementation can be declared public.